### PR TITLE
Add LibAMVW

### PR DESCRIPTION
**Pull request by andreasnoack**

### CODE REVIEW

File: L/LibAMVW/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+if [[ ${target} == *"mingw"* ]]; then
+    SLIB=dll
+    LIBDIR=bin
+elif [[ ${target} == *"darwin"* ]]; then
+    SLIB=dylib
+    LIBDIR=lib
+else
+    SLIB=so
+    LIBDIR=lib
+fi
```
Comment (giordano) [inline L/LibAMVW/build_tarballs.jl:25]: We already have all these variables: https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables-1
```suggestion
```

File: L/LibAMVW/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    doubleshift/src/DFGR.f90 \
+    doubleshift/src/DGR.f90 \
+    doubleshift/src/DMQF.f90 \
+    doubleshift/src/DNORMALPOLY.f90 \
+    doubleshift/src/DRANDPOLYJT.f90 \
+    doubleshift/src/RESCHECK.f90 \
+    doubleshift/src/balance.f90 \
+    doubleshift/src/init_random_seed.f90 \
+    doubleshift/src/turnovers/DGTO2.f90 \
+    -o $prefix/$LIBDIR/libamvwdouble.$SLIB $LBLAS
```
Comment (giordano) [inline L/LibAMVW/build_tarballs.jl:50]:
```suggestion
    -o ${libdir}/libamvwdouble.${dlext} ${LBLAS}
```

File: L/LibAMVW/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    singleshift/src/fuse.f90 \
+    singleshift/src/init_random_seed.f90 \
+    singleshift/src/modified_quadratic.f90 \
+    singleshift/src/normalpoly.f90 \
+    singleshift/src/rescheck.f90 \
+    singleshift/src/throughdiag.f90 \
+    singleshift/src/zamvw.f90 \
+    singleshift/src/zamvw2.f90 \
+    singleshift/src/turnovers/dto4.f90 \
+    -o $prefix/$LIBDIR/libamvwsingle.$SLIB $LBLAS
```
Comment (giordano) [inline L/LibAMVW/build_tarballs.jl:70]:
```suggestion
    -o ${libdir}/libamvwsingle.${dlext} $LBLAS
```

**Comment by andreasnoack** [comment]
Thanks for the comments.

**Comment by giordano** [comment]
Thanks!

**Pull request merged**